pkg/binary/envoy/runtime.go

```diff
@@ -85,6 +84,7 @@ type fetcher struct {
 
 // Runtime manages an Envoy lifecycle including fetching (if necessary) and running
 type Runtime struct {
+	binary.FetchRunner
```

Comment:
personally I don't want to abuse struct embedding since it obscures the usage.. (actually where is this used for?) Can we have a named field?

Reply:
I'll revert it.. I was trying to enforce the interface instead of duck typing, but yeah this is distracting.